gke-stateful-mysql/kubernetes/prepare-for-ha.yaml

```diff
@@ -31,6 +31,8 @@ spec:
         app: prepare-three-zone-ha
     spec:
       affinity:
+        # Tell Kubernetes to avoid scheduling a replica in a zone where there
+        # is already a replica with the label "app: prepare-three-zone-ha"
         podAntiAffinity:
```

Comment:
Let me know if I should remove this comment.
I personally found this chunk difficult to interpret, so I added a comment.
See [K8s documentation about podAntiAffinity here](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#:~:text=In%20the%20following%20example%20Deployment%20for%20the%20Redis%20cache).